paint.net-plugin-pyrochild@2020-11-21: download via aria2 failed

It looks like the getpaint.net URL for this package returns a 403 when trying to install with scoop (or any CLI tool). The download works fine in the browser; could be a user agent issue.

With aria2:
```
Installing 'paint.net-plugin-pyrochild' (2020-11-21) [64bit] from 'extras' bucket
Starting download with aria2 ...
Download: 03/11 10:50:21 [ERROR] CUID#8 - Download aborted. URI=https://forums.getpaint.net/applications/core/interface/file/attachment.php?id=18335#/Download: Download Results:                                                                                                                 d=18335
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: e60d04|ERR |       0B/s|C:/Users/xxx/scoop/cache/paint.net-plugin-pyrochild#2020-11-21#80da1f2.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

WARN  Download failed! (Error 22) HTTP response header was bad or unexpected
WARN  https://forums.getpaint.net/applications/core/interface/file/attachment.php?id=18335#/dl.zip
    referer=https://forums.getpaint.net/applications/core/interface/file/attachment.php?id=18335#/
    dir=C:\Users\xxx\scoop\cache
    out=paint.net-plugin-pyrochild#2020-11-21#80da1f2.zip

WARN  & 'C:\Users\xxx\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\xxx\scoop\cache\paint.net-plugin-pyrochild.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.6 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=32300 --continue --summary-interval=0 --auto-save-interval=1 --disable-ipv6 true
WARN
Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=paint.net-plugin-pyrochild%402020-11-21%3a+download+via+aria2+failed
Fallback to default downloader ...
The remote server returned an error: (403) Forbidden.
URL https://forums.getpaint.net/applications/core/interface/file/attachment.php?id=18335#/dl.zip is not valid
```

Without aria2:
```
Installing 'paint.net-plugin-pyrochild' (2020-11-21) [64bit] from 'extras' bucket
The remote server returned an error: (403) Forbidden.
URL https://forums.getpaint.net/applications/core/interface/file/attachment.php?id=18335#/dl.zip is not valid
```

--- a/bucket/paint.net-plugin-pyrochild.json
+++ b/bucket/paint.net-plugin-pyrochild.json
@@ -4,7 +4,7 @@
     "homepage": "https://forums.getpaint.net/topic/7291-pyrochild-plugins",
     "license": "MIT",
     "depends": "paint.net",
-    "url": "https://forums.getpaint.net/applications/core/interface/file/attachment.php?id=18335#/dl.zip",
+    "url": "https://web.archive.org/web/20240409113833if_/https://content-restricted.invisioncic.com/r125076/monthly_2020_11/pyrochild.2020-11-21_zip.fcab871e681c758ff38f832e2d05956e?response-content-disposition=attachment;%20filename=pyrochild.2020-11-21.zip&Expires=1712663913&Key-Pair-Id=K186TDY4WHSHHJ&Signature=jrCXSFGEuqMvSYdmDfqmzfo1LuAeoWS7NyoKwgqn8Y2KW0ttto~PSJkGtWpAAuOLKsDfORtdv7oxc0z1h9a3I8qfUXZMEsVd~Mzc7ntAECM5lsGdUvE2Se3MErQUS836mOGeZDb5QN~qjOf9e7L1XxxY7lRYK9HUM2zkCj-R2h-7o3n7hEhB5oWD9JZ-PlPh1ii1F5Vtae3PXvcYx-4R0-B32pZFQ5UiW7Cj6xgnMgPTAiuk3LrBlljMph8U2pEI3FXgfIVryGN1le5zXe94D0lOgSsq0iMGH3Ajd6a891DDofbPbJ6e3-WRUQMXSPqggM2Y6Dp-xNYQ3bQPr1w-Fw__#/dl.zip",
     "hash": "88261fd6eb2ebf89924aab93083d9890ef43bdb468bf6e342e355494423cf6f5",
     "pre_install": [
         "$pluginFolder = Join-Path (appdir 'paint.net' $global) 'current\\Effects'",

